api/src/main/java/org/pmiops/workbench/mail/MailServiceImpl.java

```diff
@@ -59,18 +59,42 @@
 
 @Service
 public class MailServiceImpl implements MailService {
+  private static final String AOU_ITALICS = "<i>All of Us</i>";
+  private static final String EGRESS_FORM_LINK =
+      "<a href=\"https://redcap.pmi-ops.org/surveys/?s=YJ8FJXRJD7JY4NK7\">Researcher Account Activity Confirmation</a>";
+  private static final String DOWNLOAD_FORM_LINK =
+      "<a href=\"https://redcap.pmi-ops.org/surveys/?s=YRXMJFJ97J3WMWLE\">Large Data Download Request</a>";
 
   public static final Map<EgressRemediationAction, String> EGRESS_REMEDIATION_ACTION_MAP =
       Map.of(
-          EgressRemediationAction.DISABLE_USER,
-          "Your account has been disabled pending manual review by the <i>All of Us</i> "
-              + "security team.",
           EgressRemediationAction.SUSPEND_COMPUTE,
-          "Your Workbench compute access has been temporarily suspended, and will be "
-              + "automatically restored after a brief duration.");
-  private final Provider<MandrillApi> mandrillApiProvider;
-  private final Provider<CloudStorageClient> cloudStorageClientProvider;
-  private final Provider<WorkbenchConfig> workbenchConfigProvider;
+          String.format(
```

Comment:
a substantial portion of the egress `content.html` is now here, because the contents depend on which kind of egress event it is: SUSPEND or DISABLE